modules/candidate_parameters/ajax/formHandler.php

```diff
@@ -0,0 +1,309 @@
+<?php
+
+if (isset($_POST['tab'])) {
+    $tab = $_POST['tab'];
+    if ($tab == "candidateInfo") {
+        editCandInfoFields();
+    }
+    else if ($tab == "probandInfo") {
+        editProbandInfoFields();
+    }
+    else if ($tab == "familyInfo") {
+        editFamilyInfoFields();
+    }
+    else if ($tab == "deleteFamilyMember") {
+        deleteFamilyMember();
+    }
+    else if ($tab == "participantStatus") {
+        editParticipantStatusFields();
+    }
+    else if ($tab == "consentStatus") {
+        editConsentStatusFields();
+    }
+    else {
+        header("HTTP/1.1 404 Not Found");
+        exit;
+    }
+}
+
+function editCandInfoFields() {
+    $db   =& Database::singleton();
```

Comment:
This block of code (line 30-35) is reused in all functions. Should probably go outside (e.g after line 4)